cmd/oras/pull.go

```diff
@@ -103,17 +104,38 @@ func runPull(opts pullOptions) error {
 
 	// Copy Options
 	copyOptions := oras.DefaultCopyOptions
+	configFileName, configMediaType := parseFileRef(opts.ManifestConfigRef, oras.MediaTypeUnknownConfig)
+	copyOptions.FindSuccessors = func(ctx context.Context, fetcher content.Fetcher, desc ocispec.Descriptor) ([]ocispec.Descriptor, error) {
+		successors, err := content.Successors(ctx, fetcher, desc)
+		if err != nil {
+			return nil, err
+		}
+		ret := []ocispec.Descriptor{}
+		for _, s := range successors {
+
+			if s.MediaType == configMediaType {
+				// Add annotation for manifest config
+				if s.Annotations == nil {
+					s.Annotations = make(map[string]string)
+				}
+				s.Annotations[ocispec.AnnotationTitle] = configFileName
+			}
+			if s.Annotations[ocispec.AnnotationTitle] != "" {
+				// Only copy named blobs
+				ret = append(ret, s)
```

Comment:
@shizhMSFT Do we need to print digest and media type for skipped nodes in verbose mode?

Reply:
@qweeah Wait a second. We don't even print out the digest and media type for unamed nodes, do we? Let's not do this in this PR. We should create an issue instead.

Reply:
Created issue #415